homepage/tests.py

```diff
@@ -72,3 +76,20 @@ def test_post(self):
                              target_status_code=200, fetch_redirect_response=True)
         wrong_response = self.client.post(reverse('homepage:edit', kwargs={'id': self.post.id, 'skey': self.wrong_key}))
         self.assertEqual(wrong_response.status_code, 404)
+
+class Rendering(TestCase):
+
+    def setUp(self):
+        self.input = {'post_tittle': 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa',
+                      'post_content': 'this tag will not render<a>\n\nthis is for test'}
+
+        self.input1 = {'post_tittle': 'aaaaaaaaaaa', 'post_content': '<h1>will</h1><h2>hi</h2>'}
+
+
+    def test_multiple_para_rendering_and_NO_unsafe_tags(self):
+        response = self.client.post(reverse('homepage:create'), self.input, follow=True)
+        self.assertContains(response, '<p>this tag will not render&lt;a&gt;</p>', status_code=200)
+
+    def test_TOC(self):
+        response = self.client.post(reverse('homepage:create'), self.input1, follow=True)
+        self.assertContains(response, '<ul><li>will</li></ul><ul><ul><li>hi</li></ul></ul>', status_code=200)
```

Comment:
Fix the code so it needs only 1 `ul` tag for many `li`

Reply:
rendering is not working/looking fine when doing like that